docs/tools/vdb_table/data/redis.json

```diff
@@ -32,9 +32,9 @@
     "comment": ""
   },
   "hybrid_search": {
-    "support": "",
+    "support": "full",
     "source_url": "",
```

Comment:
@tylerhutcherson can you add link to docs for the feature?

Reply:
At the moment we just have these notebook examples linked in the comment, but we are working on adding it to our documentation as well. Will ping when we have the link ready

Reply:
ok then let's add the link to the notebook into the source_url for now, which will make it easy to discover and click from the comparison table

Reply:
will do -- we should have an updated docs page ASAP and I can come back and make that update too.